src/styles/blocks/contact.scss

```diff
@@ -0,0 +1,82 @@
+//CONTACT BLOCK:
+
+.contact {
+
+  &__value-name {
+    font-size: 14px;
+    line-height: 27px;
+    color: $tracperant-text;
+
+    margin-bottom: 8px;
+  }
+  &__value {
+    font-size: 16px;
+    line-height: 27px;
+    color: $regular-text-color;
+
+    margin-bottom: 24px;
+  }
+  &__input {
+    box-sizing: border-box;
+    outline: none;
+    width: 100%;
+    height: 48px;
+
+    margin-bottom: 16px;
+    padding: 14px 17px;
+    font-size: 14px;
+    line-height: 20px;
+    color: $color-black;
+    background-color: $form-bg;
+
+    border-radius: 8px;
+    border: 2px solid transparent;
+
+    transition: border-color $transition-duration;
+
+    &::placeholder {
+      font-family: "Roboto", sans-serif;
```

Comment:
You already do it to whole page

Reply:
Yeah, but as i realized `::placeholder` pseudo doesnt inharit `font-family` so i nead to write it exactly. when i"m removing this rule font changes to somthing like **times new roman**((